### PR TITLE
docs: add explainer for non-tech

### PR DESCRIPTION
**Pull request by noelmace**
first attempt to write a "storytelling for non-technical people" following https://github.com/blindnet-io/devrel-planning/issues/51

### CODE REVIEW

File: docs/nontech-explainer.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Blindnet doesn't fit in this category, obviously. We are the new challenger who's here to disrupt the market with innovation and an outsider's perspective. We need to use weapons that fits this challenge.
+
+Then, there are the products for non-technical people, on which the brand image and market fit can be built with traditional marketings tools like ads and surveys.
+
+At blindnet, our first goal is to address developers' needs. This audience isn't sensible to traditional marketing but has high expectations for openness and collaboration.
+
+Furthermore, we work on Data Privacy, which is a security matter. And no one still believes security can be achieved by secrecy. Everybody (especially developers) is aware that computers aren't trustworthy. We need to open the hood and check how the engine works. To ensure the security software we use can guarantee security by itself, using solutions like end-to-end cryptography.
+
+At blindnet, we strongly believe this is the only way to go. That Openness and Privacy are two sides of the same coin. People's trust in us is based on facts, and nothing else.
```
[the remark this answers is not in the document]
Comment (noelmace) [reply docs/nontech-explainer.md]: Discussion moved to https://github.com/blindnet-io/communication-management/issues/86

**Comment by noelmace** [comment]
@Vuk-BN, @milstan (+ @timnicblind and @filipblnt, if you have time)

I believe we can move on and use this as a blog post as it is, as we continue the discussion in https://github.com/blindnet-io/communication-management/issues/51. So I'm asking you for a new review with this only purpose in mind.

**Comment by milstan** [comment]
Few broad comments:
- is there a link between openness and the notion (in computer logic) of Open-world assumption vs closed-world assumption? This is not necessarily to say we have to mention it in the blog (as those are tech jargon) but rather have a clear position in our mind. 

- the web is made on top  of the open-world assumption and the "anyone can say anything about anything" principle, so we have 

I'll review the text in detail later today

File: docs/nontech-explainer.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Openness in Tech: An Explainer With NO Tech Jargon
+
+> What openness means, and why we care
+
+## Why We Need Openness
+
+In its broader sense, openness is mostly defined by negation. "Being open" means to "not be" closed, as boundaries allow to define closeness more clearly.
+
+The same idea goes for the more specific sense of openness we find in IT. IT mostly took shape during the cold war with the emergence of a "closed-world discourse[^1]" emphasizing security through obscurity and global surveillance. This discourse strongly influenced our vision of IT, and continues to do so today. We still tend to see code, innovation, and information as something we need to protect. We still think we need some kind of magical secret on which we could build benefits.
```
Comment (milstan) [inline docs/nontech-explainer.md:9]: > We still think we **need** some kind of magical secret on which we could build benefits.

Maybe restructure the phrase to start with benefits, and shift from need to can 

|--> We still think we can harness benefits by holding on some kind of magical secret.

File: docs/nontech-explainer.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+But the era of secrets is long gone. Now, people expect access to all the information they need, whenever they need. Secrecy in business and technology is considered suspicious. Transparancy is key.
+
+And it goes both ways. Customers are now using social platforms on a daily basis, constantly sharing and collaborating with their peers. Here again, secrecy is a far memory, but without necessarily leading privacy to its end. The public expect a control proportionnal to the amount of data it shares. It can't accept to be passive anymore. Companies now have to offer everyone both control over their data, and opportunities to actively contribute.
```
Comment (milstan) [inline docs/nontech-explainer.md:13]: > The public expect a control proportionnal to the amount of data it shares.
I find this phrase complicated. The idea of the **public** beign the subject and then the "it shares" makes this complicated - the subject is an **it** who is very active, expects/accepts...

Maybe "the modern user expects..."

File: docs/nontech-explainer.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+- Professionnal Services, like consulting, support and training,
+- and more.
+
+## Why We Can't Afford Closeness
+
+Some companies can still manage to make money by selling proprietary software.
+
+First, there are Big Corps, like Microsoft or Oracle. Their first common denominator is a very strong and well-established brand. People can trust them because of this brand image, without needing to collaborate with them actively[^5]. Today, blindnet doesn't fit in this category. We are the new challenger who's here to disrupt the market with innovation and an outsider's perspective. We need to use weapons that fit this challenge.
+
+Then, there are the products for non-technical people, on which the brand image and market fit can be built with traditional marketings tools like ads and surveys.
```
Comment (milstan) [inline docs/nontech-explainer.md:52]: I am not sure this statement makes concensus :) i.e. that Microsoft and Oracle would agree.

Some of our, ex-Microsoft, investors might raise an eybrow.

**Review by milstan — APPROVED** [review]
Nice work.

Here are a few reactions, nothing major.

**Review by filipblnt — APPROVED** [review]

File: docs/nontech-explainer.md (hunk 2 above)
Comment (noelmace) [reply docs/nontech-explainer.md:9]: Great feedback, thx :+1:

File: docs/nontech-explainer.md (hunk 4 above)
Comment (noelmace) [reply docs/nontech-explainer.md:52]: Now that you said it, I realize I don't even agree with this myself.
Let me know if this new sentence is good enough (and grammatically correct).

File: docs/nontech-explainer.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+At blindnet, our first goal is to address developers' needs. This audience isn't sensible to traditional marketing but has high expectations for openness and collaboration.
+
+Furthermore, we work on Data Privacy, which is a security matter. And no one still believes security can be achieved by secrecy. Everybody (especially developers) is aware that computers aren't trustworthy. We need to open the hood and check how the engine works. To ensure the security software we use can guarantee security by itself, using solutions like end-to-end cryptography.
```
[the remark this answers is not in the document]
Comment (noelmace) [reply docs/nontech-explainer.md:56]: My point here is specifically to further highlight this point in the conclusion (ABA structure).
Let me know if these last changes solve the issue here.

File: docs/nontech-explainer.md (hunk 3 above)
Comment (noelmace) [reply docs/nontech-explainer.md]: I've split this sentence for more clarity, but not sure if the second part is grammatically correct and clear enough.
Please tell me what you think.

**Comment by noelmace** [comment]
It LGTM now. @milstan, @Vuk-BN (+ optionally  @m4rk055 and @timnicblind), let's see if you have any last feedback to add (with a specific focus on grammar and clarity). If not, please add an ["Approve" review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews) so we can finally publish this.

**Review by milstan — APPROVED** [review]

**Review by timnicblind — APPROVED** [review]

**Review by Vuk-BN — APPROVED** [review]

**Pull request merged**